tests/features/internal_report.feature

```diff
@@ -11,3 +13,11 @@ Scenario: Sending internal error reports
     And the event "breadcrumbs" is null
     And the event "app.type" equals "android"
     And the event "device.osName" equals "android"
+    And the event "metaData.BugsnagDiagnostics.cacheTombstone" is false
```

Comment:
There's no test to assert that this is true when cache behavior is set, or that its null when using a lower Android version.